React/Services/ReactForteService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using ReactForte.Application;
+
+namespace React.Services;
+
+internal class ReactForteService : IReactService
+{
+    private readonly IHtmlService _htmlService;
+
+    public ReactForteService(IHtmlService htmlService)
+    {
+        _htmlService = htmlService;
+    }
+
+    public async Task<IHtmlContent> RenderToStringAsync<T>(IHtmlHelper htmlHelper, string componentName, T props)
```

Comment:
as `htmlHelper` is not used in this concrete please consider injecting it via constructor to `ReactDotNetService` and change method to `Task<IHtmlContent> RenderToStringAsync<T>(string componentName, T props)`